source/_docs/configuration/remote.markdown

```diff
@@ -15,13 +15,25 @@ Remember to follow the [securing checklist](/docs/configuration/securing/) befor
 
 </div>
 
+## Setting up remote access
 <div class='note'>
-Home Assistant no longer support remote access via IP address since release 0.77, you have to use a domain name.
+Home Assistant no longer supports remote access via IP address since release 0.77, you have to use a domain name.  A Domain can be obtained cheaply from several providers. 
```

Comment:
```suggestion
Home Assistant no longer supports remote access via IP address since release 0.77, you have to use a domain name. A Domain can be obtained from several providers. 
```